sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 1.0.0b1 (Unreleased)
 
+### Breaking Changes
+
+- The `synthetic` namespace has been renamed to `simulator`, and sub-namespaces under this module have been removed
+- The `evaluate` and `evaluators` namespaces have been removed, and everything previously exposed in those modules has been added to the root namespace `azure.ai.evaluation`  
+
 ### Features Added
```

Comment:
Not related to this PR, but wonder if we should port the latest change log entry from pf-evals to here? I am referring to v0.3.3 (Upcoming)

https://github.com/microsoft/promptflow/blob/main/src/promptflow-evals/CHANGELOG.md